_docs/Salesforce-Integration.md

```diff
@@ -235,4 +235,140 @@ To integrate Salesforce with Visual Studio Code, see [Salesforce Extension Pack]
+```
+This piece of code comes directly from the localization-manager turbo component. Make sure you're using the latest code from [https://unpkg.com/@coveops/localization-manager/dist/index.min.js](https://unpkg.com/@coveops/localization-manager/dist/index.min.js), or the at least the same version (i.e. [https://unpkg.com/@coveops/localization-manager@1.0.1/dist/index.min.js](https://unpkg.com/@coveops/localization-manager@1.0.1/dist/index.min.js) for version 1.0.1).
+
+```
+CoveoLocalizationManager(LOCALES);
+```
+This final piece of code initializes the translations.
+
+Copy and paste that code for every translation you have, replacing the contents of the `LOCALES` variable with the respective translations.
+
+Finally, we will need to load those files as custom scripts in our Salesforce aura component:
+
+```
+<CoveoV2:CommunitySearch 
+    searchHub="{!v.searchHub}" 
+    name="{!v.name}"
+    debug="{!v.debug}"
+    customScripts="{!join(',',
+        $Resource.CoveoV2__searchUi + '/js/cultures/' + $Locale.language + '.js',
+        $Resource.<COMPONENT_NAME> + '/locales/' + $Locale.language + '.js',
+        $Resource.<COMPONENT_NAME> + '/index.min.js'
+    )}"
+/>
+```
+It's as simple as that!
+
+Now, if you have language codes that differ from Coveo's language codes, like Spanish, you can do the following:
+
+```
+    <aura:if isTrue="{!$Locale.language == 'es'}">
+        <CoveoV2:CommunitySearch 
+            searchHub="{!v.searchHub}" 
+            name="{!v.name}"
+            debug="{!v.debug}"
+            customScripts="{!join(',',
+                $Resource.CoveoV2__searchUi + '/js/cultures/es-es.js',
+                $Resource.<COMPONENT_NAME> + '/locales/es-es.js',
+                $Resource.<COMPONENT_NAME> + '/index.min.js'
+            )}"
+        />
+        <aura:set attribute="else">
+            <CoveoV2:CommunitySearch 
+                searchHub="{!v.searchHub}" 
+                name="{!v.name}"
+                debug="{!v.debug}"
+                customScripts="{!join(',',
+                    $Resource.CoveoV2__searchUi + '/js/cultures/' + $Locale.language + '.js',
+                    $Resource.<COMPONENT_NAME> + '/locales/' + $Locale.language + '.js',
+                    $Resource.<COMPONENT_NAME> + '/index.min.js'
+                )}"
+            />
+        </aura:set>
+    </aura:if>
+```
+
+In this instance, Salesforce defines the Spanish language code as `es` and Coveo defines it as `es-es`.
+
+For multiple conditions:
+
+```
+<aura:if isTrue="{!v.selectedVal == 'Phone'}">
```

Comment:
Not sure if the example given here is relevant to the document?

Reply:
I updated the example to be more relevant.